activefires_pp/spatiotemporal_alarm_filtering.py

```diff
@@ -189,8 +224,8 @@ def send_alarms(self, geojson_alarms, msg):
             try:
                 post_alarm(alarm['features'], self.restapi_url, self._xauth_token)
                 LOG.info('Alarm sent - status OK')
-            except HTTPError:
-                LOG.exception('Failed sending alarm!')
+            except (HTTPError, ConnectionError) as err:
+                LOG.exception('Failed sending alarm! Error: %s', str(err))
```

Comment:
The exception is going to be printed with the traceback (that `LOG.exception` will print), do we need to add it here?

Reply:
Na, you are right!